homeassistant/components/tplink/button.py

```diff
@@ -70,6 +70,8 @@ class TPLinkButtonEntityDescription(
         key="tilt_down",
         available_fn=lambda dev: dev.is_on,
     ),
+    TPLinkButtonEntityDescription(key="pair"),
+    TPLinkButtonEntityDescription(key="unpair"),
```

Comment:
Should these be marked as config?

Reply:
We usually trust upstream for categories and override them only when necessary. Here the pairing is in the config category and enabled by default, whereas the unpair is hidden under the diagnostics as it's not used that often.